File: WebsiteTinTuc.Admin/aspnet-core/src/WebsiteTinTuc.Admin.Core/Entities/CV.cs
using Abp.Domain.Entities.Auditing;
using System;
using System.ComponentModel.DataAnnotations.Schema;
using WebsiteTinTuc.Admin.Authorization.Users;

namespace WebsiteTinTuc.Admin.Entities
{
    public class CV : FullAuditedEntity<Guid>
    {
        [ForeignKey(nameof(User))]
        public long? UserId { get; set; }
        public string Link { get; set; }
        public string UserName { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string Portfolio { get; set; }
        [ForeignKey(nameof(Post))]
        public Guid PostId { get; set; }
        public virtual Post Post { get; set; }
        public virtual User User { get; set; }
    }
}

File: WebsiteTinTuc.User/WebsiteTinTuc.User/VueApp/tin-tuc-app/store/interfaces/cv.ts
import { Guid } from "guid-typescript";
import Entity from "./entity";

export default interface CV extends Entity<Guid> {
    userId?: number;
    name?: string;
    link?: string;
    userName?: string;
    email?: string;
    portfolio?: string;
    postId: Guid;
    phoneNumber?: string;
}